test/consul_cluster_with_connect_test.go

```diff
@@ -0,0 +1,26 @@
+package test
+
+import (
+	"testing"
+)
+
+/*
+// Test the example in the example-with-consul-connect folder
+func TestConsulClusterWithUbuntu16Ami(t *testing.T) {
+	t.Parallel()
+	runConsulClusterTest(t, "ubuntu16-ami", ".", "../examples/consul-ami/consul.json", "ubuntu", "")
+}*/
+
+// Test the example in the example-with-consul-connect folder
+func TestConsulConnectWithUbuntu18Ami(t *testing.T) {
+	t.Parallel()
+	runConsulClusterTest(t, "ubuntu18-ami", ".", "../examples/consul-ami/consul.json", "ubuntu", "")
```

Comment:
FYI, this isn't using the user-data scripts of the `examples/example-with-consul-connect` example. You need to modify the root example to make the user data script source configurable and pass in the path in the test.

Reply:
Fixed